test/dynamo/test_export.py

```diff
@@ -3514,6 +3514,7 @@ def forward(self, pred, x):
             [[], [], [], []],
         )
 
+    @unittest.expectedFailure  # with installing parameters on for export, this is now "fixed"
```

Comment:
Whats going on here? Maybe this is marked skip in CI because it always fails? If yes, lets send a separate PR for jus this change. Also, lets update the test to check equality rather than the failure str.

Reply:
Right, these test will now not throw assertion failure so they "fail" since the logic passes; this is sig. change (to now support globals in this manner for export), so we can follow up with the semantic change for these tests